charts/localstack/templates/deployment.yaml

```diff
@@ -157,6 +157,9 @@ spec:
           volumeMounts:
             - name: dind-storage
               mountPath: /var/lib/docker
+              {{- if ne (len .Values.persistence.extraVolumeMounts) 0 }}
```

Comment:
Unfortunately, the length check fails if the value is not set:
```bash
$ helm template ./charts/localstack
Error: template: localstack/templates/deployment.yaml:186:17: executing "localstack/templates/deployment.yaml" at <len .Values.persistence.extraVolumes>: error calling len: len of nil pointer
```
In fact, this is the default, because the new values are not set in the `values.yaml`.
Since the `values.yaml` is kind of the documentation of the available values, the new values should definitely be added there (with a bit of documentation on what they are used for).